### PR TITLE
Add note about BigInt polyfills to README

### LINKED ISSUE #309: Bug: React production build fails
### Describe the bug

Currently xmtp-js has  "@noble/secp256k1": "^1.5.2" in dependencies. In process of transpiling this package there is a known issue. This issue the same as [described](https://github.com/paulmillr/noble-ed25519/issues/23). This is another package of the same owner.

Commonly used workaround by redefine browserList to late versions of browsders reduces cover of supported browsers to 30%. 

In late releases owner fixed this code. So, maybe we could use it?
 

### Expected behavior

XMTP does not fail on React production build.

### Steps to reproduce the bug

1. Define browserList
  ```  "browserslist": {
    "production": [
      ">0.2%",
      "not dead",
      "not op_mini all"
    ],
    "development": [
      "last 1 chrome version",
      "last 1 firefox version",
      "last 1 safari version"
    ]
  }
  ```
2. npm run build
3. server -s build

### PR DESCRIPTION
**Pull request by rygine**
resolves #309

### CODE REVIEW

**Review by neekolas — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -37,30 +37,13 @@ Additional configuration is required in React environments due to the removal of
 
 ## Troubleshoot
 
-If you get into issues with Buffer and polyfills check out our [fix below](https://xmtp.org/docs/developer-quickstart#troubleshooting).
+### Buffer polyfill
 
-### Create React App
+If you get into issues with Buffer and polyfills check out our [solutions](https://xmtp.org/docs/faq#why-my-app-is-failing-saying-buffer-is-not-found).
```
Comment (jhaaaa) [inline README.md:42]:
```suggestion
If you run into issues with Buffer and polyfills, see this [solution](https://xmtp.org/docs/faq#why-my-app-is-failing-saying-buffer-is-not-found).
```

File: README.md [hunk 2]
```diff
@@ -37,30 +37,13 @@ Additional configuration is required in React environments due to the removal of
...
-
-```js
-webpack: (config, { isServer }) => {
-  if (!isServer) {
-    config.resolve.fallback.fs = false
-  }
-  return config
-}
-```
+Some older browsers do not support `BigInt` (see [compatibility list](https://caniuse.com/bigint)). Unfortunately, the way this SDK uses `BigInt` is incompatible with polyfills. To ensure that a polyfill is not added to your application bundle, update your [browserslist](https://github.com/browserslist/browserslist) configuration to exclude browsers that don't support `BigInt`.
```
Comment (jhaaaa) [inline README.md:46]:
```suggestion
This SDK uses `BigInt` in a way that's incompatible with polyfills. To ensure that a polyfill isn't added to your application bundle, update your [browserslist](https://github.com/browserslist/browserslist) configuration to exclude browsers that don't support `BigInt`.

For the list of browsers that don't support `BigInt`, see this [compatibility list](https://caniuse.com/bigint).
```

**Review by jhaaaa — APPROVED** [review]
Thank you for this and for the ping @rygine - so thoughtful of you! Made a couple of suggestions - please do with them what you will!

File: README.md (hunk 1 above)
Comment (rygine) [reply README.md]: BTW, we may want to update this sentence on our docs site to match. first sentence in the linked section: https://xmtp.org/docs/faq#why-my-app-is-failing-saying-buffer-is-not-found.

**Pull request merged**